docs/rfc/block-registration.md

```diff
@@ -337,8 +354,8 @@ wp.blocks.registerBlockType( 'my-block/name', {
 		// Save definition goes here.
 	},
 	supports: {
-		html: false
-	}
+		html: false,
```

Comment:
Should we also remove `supports` from this example?

Reply:
Yes, we should. I missed it.

By the way, I accidentaly enabled quite aggressive Visual Studio Code formatting for Markdown files.